### ISSUE TITLE
Support the use of the third-party KDNN library

### ISSUE THREAD

**Issue by amaniyadunia** [tensorflow/tensorflow#118587]
KDNN provides high-performance implementations of a series of functions. We expect TensorFlow to be able to call KDNN to improve the execution performance of certain computational graphs.
Currently, we have completed the replacement and verification of the sigmoid operator based on version 2.15.0. This is the relevant code: https://github.com/amaniyadunia/tensorflow/commit/7f28db941e4d4a7bf3fa3cfad5e0244bcef4a677.
We have implemented a custom operator that calls the KDNN function and replaces it during the graph compilation stage using the remapper mechanism, thereby achieving the goal of improving performance by using the KDNN algorithm library.
Subsequently, plans are also in place to complete the integration of Matmul, floormod, Softmax, concat, and eisum operators..

We conducted the test in the following environment:
**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04):openEuler 24.03 (LTS-SP3)
- TensorFlow installed from (source or binary):source 
- TensorFlow version (or github SHA if from source):v2.15

**Comment by shivvaam0001** [comment]
Hi @amaniyadunia,



Thank you for taking the time to report this and for your patience.
TFLite has transitioned to LiteRT, Google's primary runtime for on-device AI. All active development, feature updates, and performance enhancements have shifted exclusively to LiteRT.
We encourage you to try reproducing the issue using LiteRT. If you still encounter this issue, please open a new issue with the updated details in the [LiteRT Repository](https://github.com/google-ai-edge/LiteRT).
For more information and step-by-step guidance on updating your project, please refer to the [LiteRT Migration Guide](https://developers.google.com/edge/litert/migration).

Thank you again for your support and understanding!
